lib/presentation/extensions/send_file_extension.dart

```diff
@@ -65,7 +67,30 @@ extension SendFileExtension on Room {
     final tempEncryptedFile =
         await File('${tempDir.path}/$formattedDateTime${fileInfo.fileName}')
             .create();
+    final tempThumbnailFile = await File(
+      '${tempDir.path}/$formattedDateTime${fileInfo.fileName}_thumbnail.jpg',
+    ).create();
+    final tempEncryptedThumbnailFile = await File(
+      '${tempDir.path}/$formattedDateTime${fileInfo.fileName}_encrypted_thumbnail',
+    ).create();
+
+    // computing the thumbnail in case we can
+    if (fileInfo is ImageFileInfo &&
+        (thumbnail == null || shrinkImageMaxDimension != null)) {
+      fakeImageEvent.rooms!.join!.values.first.timeline!.events!.first
+              .unsigned![fileSendingStatusKey] =
+          FileSendingStatus.generatingThumbnail.name;
```

Comment:
Too confusing for me. Should separate variables to make it easier to understand

Reply:
I understand your confusion
but the `fakeImageEvent` has many of the params dept
it is hard to update normally
whole of the app do the samething
We will refactor it in the future